contrib/babelfishpg_common/src/spatialtypes.c

```diff
@@ -67,134 +204,1212 @@ load_functions()
     
     if (lwgeom_x_p == NULL)
         lwgeom_x_p = (lwgeom_x_t) load_external_function("$libdir/postgis-3", "LWGEOM_x_point", true, NULL);
+
+    if (lwgeom_from_text_p == NULL)
+        lwgeom_from_text_p = (lwgeom_from_text_t) load_external_function("$libdir/postgis-3", "LWGEOM_from_text", true, NULL);
+
+    if (lwgeom_from_bytea_p == NULL)
+        lwgeom_from_bytea_p = (lwgeom_from_bytea_t) load_external_function("$libdir/postgis-3", "LWGEOM_from_bytea", true, NULL);
+
+    if (lwgeom_to_bytea_p == NULL)
+        lwgeom_to_bytea_p = (lwgeom_to_bytea_t) load_external_function("$libdir/postgis-3", "LWGEOM_to_bytea", true, NULL);
+
+    if (st_flipcoordinates_p == NULL)
+        st_flipcoordinates_p = (st_flipcoordinates_t) load_external_function("$libdir/postgis-3", "ST_FlipCoordinates", true, NULL);
+
+    if (st_point_p == NULL)
+        st_point_p = (st_point_t) load_external_function("$libdir/postgis-3", "ST_Point", true, NULL);
```

Comment:
Rather than checking `NULL` for every static variable, can we maintain a single static variable like `postgis_resources_initialized` and check for:
```
if (postgis_resources_initialized)
   return;

/* else continue the execution */
```